main.go

```diff
@@ -5,62 +5,125 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"net/http"
 	"os"
 	"regexp"
 	"strings"
-	"time"
 )
 
 func main() {
-	fmt.Printf("Please pick a word and then I will tell you how often is shows up in each of the following Shakespeare plays: The Tempest, Hamlet, King Lear, Macbeth, and Romeo and Juliet. Please pick a word...")
-	Scanner := bufio.NewScanner(os.Stdin)
-	Scanner.Scan()
-	userInputWord := Scanner.Text()
-	userInputLowercase := strings.ToLower(userInputWord)
+	log.Fatal(http.ListenAndServe(":8080", http.FileServer(http.Dir("static"))))
 
-	timeAtStart := time.Now()
+	fmt.Println("Hello and welcome to the Shakespeare Scanner.\n This is a tool which allows you to search a Shakespeare play for a word\n and it will tell you both how many times it shows up, as well as where it shows up.\n")
+	for {
+		fmt.Println("Please type the title of the play you would like to search for your word...")
+		Scanner := bufio.NewScanner(os.Stdin)
+		Scanner.Scan()
+		userPlayChoice := strings.ToLower(Scanner.Text())
+		fmt.Printf("Great, so you want to search in %s correct?\nPlease type yes or no.\n", userPlayChoice)
+		Scanner.Scan()
+		confirmation := strings.ToLower((Scanner.Text()))
+		if confirmation == "yes" {
+			fmt.Printf("Great! Please tell me what word you would like to search for in %s?\n", userPlayChoice)
+		} else if confirmation == "no" {
+			fmt.Println("Ok sorry about that. Please tell me again what play you are looking for...")
+			play2ndTry := bufio.NewScanner(os.Stdin)
+			play2ndTry.Scan()
+			confirmation := strings.ToLower((play2ndTry.Text()))
+			fmt.Printf("Let's try this again, is the play you're interested in %s?\nPlease type yes or no.\n", confirmation)
+			Scanner := bufio.NewScanner(os.Stdin)
+			Scanner.Scan()
+			finalCheck := strings.ToLower((Scanner.Text()))
+			if finalCheck == "yes" {
+				userPlayChoice := confirmation
+				fmt.Printf("Great, now you can tell me what word you want to look for in %s?\n", userPlayChoice)
+			} else if finalCheck == "no" {
+				fmt.Println("Sorry I'm obviously having a bad day. Please try again later. Goodbye!\n")
+				break
+			}
+		}
+		Scanner = bufio.NewScanner(os.Stdin)
+		Scanner.Scan()
+		finalWord := strings.ToLower((Scanner.Text()))
+		fmt.Printf("Ok so just to confirm the word you want to search is %s.\n Now searching...\n", finalWord)
+		getPlay, err := ioutil.ReadFile(userPlayChoice + ".txt")
+		if err != nil {
+			fmt.Println("Sorry we do not currently have that play in our database, please try another play.\n")
+			return
```

Comment:
Any reason why you use `return` in some places and `break` in others to exit the loop?

Reply:
Thank you for pointing this out. I went back and made changes to rectify my use of "return", "break" and "continue" so they are used correctly.